sql/redshift/001_initial.schema.up.sql

```diff
@@ -0,0 +1,39 @@
+BEGIN;
+CREATE TABLE rpc_endpoint_requests(
+    id varchar(128) not null distkey,
+    received_at timestamptz default sysdate,
```

Comment:
should this really have `default sysdate` / isn't this supposed to be set by the application?

Reply:
yes it is. but default sytemdate.